docs/deployment/README.md

```diff
@@ -147,292 +127,288 @@ These resources only need to be created once per account.
+  {
+    "arn" = "arn:aws:cloudwatch:us-east-1:12345:alarm:prefix-es-vpc-NodesLowAlarm"
+    "name" = "prefix-es-vpc-NodesLowAlarm"
+  },
+  # ... more alarms ...
+]
+elasticsearch_domain_arn = arn:aws:es:us-east-1:12345:domain/prefix-es-vpc
+elasticsearch_hostname = vpc-prefix-es-vpc-abcdef.us-east-1.es.amazonaws.com
+elasticsearch_security_group_id = sg-12345
 ```
 
-You can monitor the progess of the stack deployment from the [AWS CloudFormation Console](https://console.aws.amazon.com/cloudformation/home); this step takes a few minutes.
+Your data persistence resources are now deployed.
 
-A successful completion will result in output similar to:
+## Deploy the Cumulus Message Adapter layer
 
-```bash
-  $ DEPLOYMENT=<cumulus-deployment-name> \
-      AWS_REGION=<region> \ # e.g. us-east-1
-      AWS_PROFILE=<profile> \
-      npm run deploy-app
+**Note:**: If you are deploying in an NGAP environment, you should be able to use the existing Cumulus Message Adapter layer deployed in your environment.
 
-  Nested templates are found!
+The [Cumulus Message Adapter (CMA)](./../workflows/input_output.md#cumulus-message-adapter) is necessary for interpreting the input and output of Cumulus workflow steps. The CMA is now integrated with Cumulus workflow steps as a Lambda layer.
 
-  Compiling nested template for CumulusApiDistribution
-  Zipping app/build/cumulus_api/0000UUID-ApiDistribution.zip for ApiDistribution
-  Uploaded: s3://<prefix>-internal/<prefix>-cumulus/lambdas/0000UUID-ApiDistribution.zip
-  Template saved to app/CumulusApiDistribution.yml
-  Uploaded: s3://<prefix>-internal/<prefix>-cumulus/CumulusApiDistribution.yml
+PLACEHOLDER FOR DEPLOYMENT STEPS
 
-  Compiling nested template for CumulusApiBackend
-  Zipping app/build/cumulus_api/0000UUID-ApiEndpoints.zip for ApiEndpoints
-  Uploaded: s3://<prefix>-internal/<prefix>-cumulus/0000UUID-ApiEndpoints.zip
-  Template saved to app/CumulusApiBackend.yml
-  Uploaded: s3://<prefix>-internal/<prefix>-cumulus/CumulusApiBackend.yml
+Make sure to copy the ARN of the deployed layer, as it will be used to configure the `cumulus-tf` deployment in the next step.
 
-  Uploaded: s3://<prefix>-internal/<prefix>-cumulus/lambdas/0000UUID-HelloWorld.zip
-  Uploaded: s3://<prefix>-internal/<prefix>-cumulus/lambdas/0000UUID-sqs2sf.zip
-  Uploaded: s3://<prefix>-internal/<prefix>-cumulus/lambdas/0000UUID-KinesisOutboundEventLogger.zip
+## Configure and deploy the `cumulus-tf` root module
 
-  Generating keys. It might take a few seconds!
-  Keys Generated
-  keys uploaded to S3
+These steps should be executed in the `cumulus-tf` directory of the template repo that was cloned previously.
 
-  Template saved to app/cloudformation.yml
-  Uploaded: s3://<prefix>-internal/<prefix>-cumulus/cloudformation.yml
-  Waiting for the CF operation to complete
-  CF operation is in state of CREATE_COMPLETE
+**Note:** When deploying to NGAP, these steps should be performed using `NGAPShNonProd` credentials.
```

Comment:
Does NGAP/do we  have a wiki page with 'how to NGAP' somewhere that we could link to here?   This seems like the sort of thing that we might want to reference whatever docs there are on deployment requirements on the ED wiki.

Reply:
Removed in 1a6bf32d580338ee3f820b39fa749a94e2849662 as this is apparently no longer true